components/Tabs/Tabs.js

```diff
@@ -2,23 +2,28 @@
 class TabItem {
   constructor(element) {
     // attach dom element to object. Example in Tabs class
+    this.element = element;
   }
 
   select() {
     // should use classList
+    this.element.classList.add('Tabs__item-selected');
   }
 
   deselect() {
     // should use classList
+    this.element.classList.remove('Tabs__item-selected');
   }
 }
 
 class TabLink {
   constructor(element, parent) {
-    this.element;// attach dom element to object
-    this.tabs;// attach parent to object
-    this.tabItem;// assign this to the associated tab using the parent's "getTab" method by passing it the correct data
+    this.element = element; // attach dom element to object
```

Comment:
I would suggest to to use the same comment styling throughout the project, you have some above others on the side.